pkg/util/os/os_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package os
+
+import (
+	"testing"
+)
+
+func Test_environment_to_slice_with_nil_parameter_returns_empty_slice(t *testing.T) {
+	testEnv := EnvironmentToSlice(nil)
+
+	if len(testEnv) != 0 {
+		t.Fatal("The returned environment should be empty.")
+	}
+}
+
+func Test_environment_to_slice_with_empty_map_returns_empty_slice(t *testing.T) {
+	testEnv := EnvironmentToSlice(make(map[string]string))
+
+	if len(testEnv) != 0 {
+		t.Fatal("The returned environment should be empty.")
+	}
+}
+
+func Test_environment_to_slice(t *testing.T) {
```

Comment:
It's a test for the env helper methods which are in os.go. I guess it depends on what you think one should and should not put in there.